kolibri/core/auth/tasks.py

```diff
@@ -353,6 +353,7 @@ def peerfacilitysync(command, **kwargs):
 
 class PeerFacilityImportJobValidator(PeerFacilitySyncJobValidator):
     facility = HexOnlyUUIDField()
+    facility_name = serializers.CharField()
```

Comment:
```suggestion
    facility_name = serializers.CharField(default="")
```

This will allow the existing tests to pass, and for this to continue to work even in the case that the frontend fails to pass this information.